examples/soundcloud.amp.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Soundcloud examples</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script async custom-element="amp-soundcloud" src="/dist/v0/amp-soundcloud-0.1.max.js"></script>
```

Comment:
I think I'm a bit lost. I don't see an `amp.soundcloud-0.1.js` file in `/dist/`. I just see a `.max.js` and `.max.js.map`.